system/jlib/jqueue.tpp

```diff
@@ -74,36 +79,26 @@ public:
     }
     inline void enqueue(BASE *e)
     {
-        if (ALLOWNULLS || e) {
-            if (num==max) 
+        if (ALLOWNULLS || e)
+        {
+            if (unlikely(num==max))
                 expand();
-            if (num==0) {
```

Comment:
Probably doesn't matter, but this does mean that a queue which becomes empty will not have the next entry pushed to be in position 0, where previously it would have been...

Reply:
I agree, but I think that is unnecessary.